spec/jobs/fail_ect_induction_job_spec.rb

```diff
@@ -1,16 +1,19 @@
 RSpec.describe FailECTInductionJob, type: :job do
   let(:teacher) { FactoryBot.create(:teacher) }
   let(:trn) { teacher.trn }
-  let(:start_date) { "2023-11-13" }
-  let(:completed_date) { "2024-01-13" }
+  let(:start_date) { Date.parse("2023-11-13") }
+  let(:completed_date) { Date.parse("2024-01-13") }
```

Comment:
The API client expects date objects, not strings

Reply:
Not sure how this was passing earlier if it was passing in strings 🤔

Reply:
Stubbing!